cmd/minikube/cmd/node_add.go

```diff
@@ -54,6 +55,11 @@ var nodeAddCmd = &cobra.Command{
 			KubernetesVersion: cc.KubernetesConfig.KubernetesVersion,
 		}
 
+		// Make sure to decrease the default amount of memory we use per VM if this is the first worker node
+		if len(cc.Nodes) == 1 && viper.GetString(memory) == "" {
```

Comment:
is this supposed to be >1 ?

Reply:
no, if cc.Nodes == 1 then the node we're currently adding is the first worker, so we need to change the memory allocation